eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -22,12 +22,18 @@
     <UsagePattern IdentityGlob="Microsoft.NETCore.App.Ref/7.0*" />
 
      <!-- This is upgraded to latest version in full source-build and can be baselined for repo build -->
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Configuration*/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.DependencyInjection/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Logging*/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Options.ConfigurationExtension*/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Options/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Primitives/8.0*" />
     <UsagePattern IdentityGlob="System.Composition*/7.0*" />
     <UsagePattern IdentityGlob="System.Configuration.ConfigurationManager/7.0*" />
+    <UsagePattern IdentityGlob="System.Diagnostics.DiagnosticSource/8.0*" />
     <UsagePattern IdentityGlob="System.Diagnostics.EventLog/7.0*" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/7.0*" />
-    <UsagePattern IdentityGlob="Microsoft.Extensions.Configuration*/7.0*" />
-    <UsagePattern IdentityGlob="Microsoft.Extensions.Logging*/7.0*" />
-    <UsagePattern IdentityGlob="Microsoft.Extensions.Options.ConfigurationExtension*/7.0*" />
+    <UsagePattern IdentityGlob="System.Text.Encodings.Web/8.0*" />
+    <UsagePattern IdentityGlob="System.Text.Json/8.0*" />
```

Comment:
I realized that these new package entries in this file will also require corresponding `PackageVersion` entries in `eng/Directory.Packages.props`. They should reference the corresponding version property from `eng/Versions.props`.

Reply:
All the packages here have corresponding versions in eng/versions.props.

Reply:
My comment doesn't seem to be addressed. For example, `Microsoft.Extensions.Configuration` is missing from `eng/Directory.Packages.props`.

Reply:
> My comment doesn't seem to be addressed. For example, `Microsoft.Extensions.Configuration` is missing from `eng/Directory.Packages.props`.

Oh I think I mis-understand your idea here. added